src/encoded/static/components/matrix_brain.js

```diff
@@ -380,13 +380,22 @@ const convertContextToDataTable = (context, diseaseList, diseaseGroupIndex = 0)
     rowKeys.push(`header${diseaseGroupIndex}`);
 
     diseaseList?.forEach((diseaseInfo) => {
-        const disease = Object.keys(diseaseInfo)[0];
+        const disease = Object.keys(diseaseInfo)[0]?.trim();
         const rowData = diseaseInfo[disease] || [];
         const rowLength = rowData[0]?.rowContent?.length || 1;
         const diseaseColor = rowData[0]?.diseaseColor || NO_DISEASE_CSS_CLASS;
-        const replicateUrlPart = disease === NO_DISEASE_LABEL
-            ? DISEASES.map((d) => `&replicates.library.biosample.disease_term_name!=${d.trim()}`).join('')
-            : disease.split('and').map((d) => `&replicates.library.biosample.disease_term_name=${d.trim()}`).join('');
+        let replicateUrlPart;
+
+        if (disease === NO_DISEASE_LABEL) {
+            replicateUrlPart = DISEASES.map((d) => `&replicates.library.biosample.disease_term_name!=${d}`).join('');
+        } else if (disease.includes('and')) {
```

Comment:
`disease` could, at least in theory, be undefined here because of the optional chaining on 383, and could crash here.